cms/static/js/views/pages/container.js

```diff
@@ -647,9 +745,10 @@ function($, _, Backbone, gettext, BasePage,
             event.preventDefault();
             try {
                 if (this.options.isIframeEmbed) {
-                    window.parent.postMessage(
+                    return window.parent.postMessage(
                         {
                             type: 'duplicateXBlock',
+                            message: 'Duplicate the XBlock',
                             payload: {}
```

Comment:
I think it's better to put the XBlock usageId in the `payload` of this event, instead of expecting the MFE to keep track of the "current" XBlock.

(Same for all other events involving a single XBlock)

Reply:
[inform] Implemented in follow-up commit: https://github.com/openedx/edx-platform/pull/35694/commits/8d5b7e3bb77cf48352f1e7864636ac05b31810e1#diff-1b7855448b6086c2a313e0b9ecff179e991106aac52648d880f6f9d551e64577R680